src/core/modules/StarWrapper.cpp

```diff
@@ -476,14 +477,14 @@ QString StarWrapper1::getInfoString(const StelCore *core, const InfoStringGroup&
 QVariantMap StarWrapper1::getInfoMap(const StelCore *core) const
 {
 	QVariantMap map = StelObject::getInfoMap(core);
-	int64_t star_id;
+	StarId star_id;
 	if (s->getHip())
 	{
-		star_id = static_cast<int64_t>(s->getHip());
+		star_id = static_cast<StarId>(s->getHip());
 	}
 	else
 	{
-		star_id = s->getGaia();
+		star_id = static_cast<StarId>(s->getGaia());
```

Comment:
Shouldn't `getGaia` (and `getHip`) return a `StarId`?

Reply:
fixed